pkg/generic/map_test/idl/example.thrift

```diff
@@ -32,7 +40,7 @@ struct A {
 }
 
 service ExampleService {
-    ExampleResp ExampleMethod(1: ExampleReq req)throws(1: Exception err),
+    ExampleReq ExampleMethod(1: ExampleReq req)throws(1: Exception err),
```

Comment:
why response name changed to "Req" ?

Reply:
Because we just want to test to check if the writers write the correct values even when there is a nil value in request like [here](https://github.com/cloudwego/kitex/pull/718/files#diff-89f634f936a8f2b7cafa6fc4ade8c5d3c0a9219ba293e4b63969af8e4ef9f91bR92-R113).
So we need `ExampleReq` as response.